service/assessment/assessment.go

```diff
@@ -318,15 +325,19 @@ func (svc *Service) handleEvidence(ev *evidence.Evidence, resourceId string) (er
 		return newError
 	}
 
-	// Get Evidence Store stream
-	channelEvidenceStore, err := svc.evidenceStoreStreams.GetStream(svc.evidenceStoreAddress.target, "Evidence Store", svc.initEvidenceStoreStream, svc.evidenceStoreAddress.opts...)
-	if err != nil {
-		err = fmt.Errorf("could not get stream to evidence store (%s): %w", svc.evidenceStoreAddress.target, err)
-		log.Error(err)
+	// Send evidence via Evidence Store stream if hasEvidenceStore is true
+	if svc.hasEvidenceStore {
+		// Get Evidence Store stream
+		channelEvidenceStore, err := svc.evidenceStoreStreams.GetStream(svc.evidenceStoreAddress.target, "Evidence Store", svc.initEvidenceStoreStream, svc.evidenceStoreAddress.opts...)
+		if err != nil {
+			err = fmt.Errorf("could not get stream to evidence store (%s): %w", svc.evidenceStoreAddress.target, err)
+			log.Error(err)
 
-		go svc.informHooks(nil, err)
+			go svc.informHooks(nil, err)
```

Comment:
formatting

Reply:
The indentation is right due to he new `if`